gst/nnstreamer/tensor_filter/tensor_filter_common.c

```diff
@@ -800,12 +909,18 @@ gst_tensor_filter_common_set_property (GstTensorFilterPrivate * priv,
         }
 
         prop->input_meta.num_tensors = num_types;
+      } else if (value) {
+        /** Update the properties */
+        if (GST_TF_FW_V0 (priv->fw)) {
+          /* Once configured, it cannot be changed in runtime */
+          g_assert (!prop->input_configured);
+        } else {
+          /** TODO: update the properties */
```

Comment:
You'll need to re-negotiate here, too. I'm not sure how that can be accomplished.

Reply:
Re-negotiation can be done by sending a GST_EVENT_RECONFIGURE event upstream ([link](https://gstreamer.freedesktop.org/documentation/plugin-development/advanced/negotiation.html?gi-language=c#upstream-caps-renegotiation)).

Yes, this is removed for now. Re-negotiation can be supported later.